src/py-opentimelineio/opentimelineio/adapters/otio_json.py

```diff
@@ -46,13 +81,39 @@ def write_to_string(input_otio, indent=4):
         indent (int): number of spaces for each json indentation level. Use\
             -1 for no indentation or newlines.
 
+    If target_schema_versions is None and the environment variable
+    "OTIO_DEFAULT_TARGET_VERSION_FAMILY_LABEL" is set, will read a map out of
+    that for downgrade target.  The variable should be of the form
+    FAMILY:LABEL, for example "MYSTUDIO:JUNE2022".
+
     Returns:
         str: A json serialized string representation
-    """
-    return core.serialize_json_to_string(input_otio, indent)
 
+    Raises:
+        otio.exceptions.InvalidEnvironmentVariableError: if there is a problem
+        with the default environment variable
+        "OTIO_DEFAULT_TARGET_VERSION_FAMILY_LABEL".
```

Comment:
```suggestion
    :raises otio.exceptions.InvalidEnvironmentVariableError: if there is a problem
        with the default environment variable
        ``OTIO_DEFAULT_TARGET_VERSION_FAMILY_LABEL``.
```

See https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists for the syntax.

Or is it better to keep the current style because it's easier to display in the documentation?

Reply:
@reinecke you have your ear to the ground with this stuff - is it better to add markup for the  sphinx docs?

Reply:
I cleaned up the docs in core/__init__.py and from the python wrapped modules.  Doing a consistency pass on the adapter docs is probably a good idea but lets do it as a later thing.  Good catch though.  Thanks!